object-manipulation-1/src/get-keys.js

```diff
@@ -0,0 +1,10 @@
+/* exported getKeys */
+function getKeys(object) {
+  var arr = [];
+  for (var key in object) {
+    if (key in object) {
```

Comment:
Why is this conditional necessary? If there's a value for `key`, then that means it is a key in the object, right? 🤔